### ISSUE TITLE
More lenient comment parsing for `swiftformat:` directives

### ISSUE THREAD

**Issue by lhunath** [nicklockwood/SwiftFormat#1027]
Currently, `swiftformat:` directives in comments are only honoured if they follow a strict pattern and are used, for instance, as such:

```
// swiftformat:disable trailingClosures
```

It may be desirable to mix these comments with information or with markers such as FIXME/TODO. The following no longer works:

```
// FIXME: swiftformat:disable trailingClosures, https://github.com/nicklockwood/SwiftFormat/issues/1026
```

A work-around is to put the FIXME token at the end:

```
// swiftformat:disable trailingClosures, FIXME: https://github.com/nicklockwood/SwiftFormat/issues/1026
```

Unfortunately, this is not ideal since tools that parse FIXMEs now no longer have the full desired context.

It should be noted that `swiftlint` supports the following:

```
// FIXME: swiftlint:disable:next force_cast - make extension conditional on Tabbed presenters.
```

**Comment by lhunath** [comment]
Plenty of things I *can* do, but this request is essentially to consider the lenience that would allow me to tie more information into my marks & to synchronize with the capabilities of swiftlint.

**Comment by nicklockwood** [comment]
@lhunath fixed in 0.48.15
